apps/src/code-studio/pd/workshop_enrollment/enroll_form.tsx

```diff
@@ -505,209 +534,262 @@ export default function EnrollForm(props: EnrollFormProps) {
+            errorMessage={formErrors.first_name}
+            className={getRequiredStyles('first_name')}
+          />
+          <TextField
+            id="last_name"
+            name="last_name"
+            label="Last Name"
+            onChange={e =>
+              handleChange({
+                last_name: e.target.value,
+              })
+            }
+            value={formState.last_name}
+            errorMessage={formErrors.last_name}
+            className={getRequiredStyles('last_name')}
+          />
+          <TextField
+            id="email"
+            name="email"
+            label="Email Address"
+            onChange={e =>
+              handleChange({
+                email: e.target.value,
+              })
+            }
+            value={formState.email}
+            title={
+              props.email ? 'Email can be changed in account settings' : ''
+            }
+            errorMessage={formErrors.email}
+            className={getRequiredStyles('email')}
+          />
+          {!props.email && (
+            <TextField
+              id="confirm_email"
+              name="confirm_email"
+              label="Confirm Email Address"
+              onChange={e =>
+                handleChange({
+                  confirm_email: e.target.value,
+                })
               }
-              errorMessage={formErrors.first_name}
+              value={formState.confirm_email}
+              errorMessage={formErrors.confirm_email}
+              className={getRequiredStyles('confirm_email')}
             />
-            <FieldGroup
-              id="last_name"
-              label="Last Name"
-              type="text"
-              required={true}
-              onChange={handleChange}
-              validationState={
-                Object.prototype.hasOwnProperty.call(formErrors, 'last_name')
-                  ? VALIDATION_STATE_ERROR
-                  : null
-              }
-              errorMessage={formErrors.last_name}
+          )}
+          <div className={styles.school_info_container}>
+            <SchoolDataInputs
+              includeHeaders={false}
+              containerClassName={styles.school_info_required}
+              {...schoolInfo}
             />
-            <FieldGroup
-              id="email"
-              label="Email Address"
-              type="text"
-              required={true}
-              onChange={handleChange}
-              defaultValue={props.email}
-              title={
-                props.email ? 'Email can be changed in account settings' : ''
-              }
-              validationState={
-                Object.prototype.hasOwnProperty.call(formErrors, 'email')
-                  ? VALIDATION_STATE_ERROR
-                  : null
-              }
-              errorMessage={formErrors.email}
+            <Label
+              errorMessage={formErrors.school_info}
+              htmlFor="school_info"
```

Comment:
I _think_ this needs to be the ID of the input -- can you export the ID from the component file?